wordpress-mobile/WordPress-Android
# Private & Pending Pages

Fixes #8313. This PR adds support for pages with `Private` and `Pending review` status. The new design includes support for 2 tags under the page title.

To test:
1. Open a site with pages that have `Private` or `Pending review` status
2. Navigate to Site pages
3. Notice the app doesn't crash and you can see the pages marked with the correct status

Note: I've discovered a bug when `Private` or `Pending review` status isn't saved when the Back button is pressed instead of Update in the editor. This is unrelated and will be fixed in a different PR.

Also, I've temporarily disabled the PPP unit tests because they keep failing in Travis even though they are correct. This is already being addressed by #8303.
